src/framework/vst/internal/vstinstancesregister.h

```diff
@@ -72,12 +71,41 @@ class VstInstancesRegister : public IVstInstancesRegister, public async::Asyncab
 private:
     mutable std::mutex m_mutex;
 
-    using FxPluginKey = std::pair<muse::audio::AudioResourceId, muse::audio::AudioFxChainOrder>;
-    using VstFxInstancesMap = std::map<FxPluginKey, IVstInstancePtr>;
+    enum class Type {
```

Comment:
We can use muse::audioplugins::AudioPluginType here

Reply:
this is just an internal private implementation of this class... it can be done in different ways, we can add your own service types here depending on the need